tests/core/test_evaluation.py

```diff
@@ -290,3 +290,34 @@ async def test_retrieval_intent_wrong_prediction(
 
     # check if the predicted entry contains full retrieval intent
     assert "# predicted: chitchat/ask_name" in failed_stories
+
+
+async def test_e2e_with_entity_evaluation(e2e_bot_agent: Agent, tmpdir: Path):
```

Comment:
`tmpdir` has a different (weird) type
```suggestion
async def test_e2e_with_entity_evaluation(e2e_bot_agent: Agent, tmp_path: Path):
```